src/components/usertask-component/form-fields/boolean-form-field.tsx

```diff
@@ -0,0 +1,39 @@
+/* tslint:disable */
+import {Component, Prop, State} from '@stencil/core';
+import {DataModels} from '@process-engine/consumer_api_contracts';
+import {IFormField} from './iform_field';
+
+@Component({
+  tag: 'boolean-form-field',
+  styleUrl: 'boolean-form-field.css',
+  shadow: true,
+})
+
+export class BooleanFormField implements IFormField {
+
+  formField: DataModels.UserTasks.UserTaskFormField;
+
+  public get name(): string {
+    return this.formField.id;
+  }
+
+  @State() public value: boolean;
+
+  // tslint:disable-next-line:typedef
+  public componentWillLoad() {
+    this.value = this.formField.defaultValue === 'true';
+  }
+
+  render() {
```

Comment:
Public? Private? Typedef?

```suggestion
  public render(): HTMLElement | any {
```